### DISCUSSION TITLE
Sort grid by date property

### DISCUSSION THREAD

**Discussion by XNoobMaster69** [AppFlowy-IO/AppFlowy#2668] [Ideas]
I just started using Appflowy and created a grid with date as a property. Intuitively, I tried to sort the items by the date property, which was not available. It would be a great help if that was possible.

**Comment by i5hi** [comment]
Had the same issue today.

**Comment by annieappflowy** [comment]
Supporting it soon
Thanks @XNoobMaster69  and @i5hi for the feedback.
